lib/elixir/lib/kernel.ex

```diff
@@ -3823,10 +3832,8 @@ defmodule Kernel do
         end
       end
 
-  By calling `use/2`, a hook called `__using__/1` will be invoked in
-  `ExUnit.Case` which will then do the proper setup.
-
-  Simply put, `use/2` translates to:
+  In this example, `ExUnit.Case.__using/1` is called with the keyword list
```

Comment:
I can write `__using__/1` one thousand times, and I will still miss the `__` after it 😞